hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandlersModule.java

```diff
@@ -55,14 +55,14 @@ static TransactionHandlers provideTransactionHandlers(
                 contractComponent.contractSystemDeleteHandler(),
                 contractComponent.contractSystemUndeleteHandler(),
                 contractComponent.etherumTransactionHandler(),
-                tokenComponent.cryptoCreateHandler(),
```

Comment:
We should probably keep the old names. I see no benefit in renaming everything at this point.